src/components/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Img from '../images/space_travelers.png';
+
+const Navbar = () => (
+  <nav>
+    <div className="nav_container">
+      <div className="image_container">
+        <img alt="header logo" src={`${Img}`} />
+        <h1>Space Travlers&apos; Hub</h1>
+      </div>
+      <ul className="nav_links">
+        <Link to="/"><li className="page_links">Rockets</li></Link>
+        <Link to="/Missions"><li className="page_links">Missions</li></Link>
+        <div className="line_end" />
+        <Link to="/MyProfile"><li className="page_links">My profile</li></Link>
+      </ul>
```

Comment:
As required, `<NavLink />` should be used for navigation links, Kindly fix this.

![image](https://user-images.githubusercontent.com/66112760/163470700-555d384d-33f9-40e3-ad29-dded13e5244a.png)